### ISSUE TITLE
GLTF children missing from hierarchy

### ISSUE THREAD

**Issue by ncomendant** [rewin123/space_editor#150]
When attempting to open this [GLTF file](https://poly.pizza/m/bN9Oz3niNm), it will appear in Game View and World Inspector correctly, but the hierarchy is missing children.

Here is a screenshot:
![image](https://github.com/rewin123/space_editor/assets/3222456/4faf2b12-87f1-412b-95fe-6c9752aeeaaf)

You can see all the children with Pbr Mesh are missing. They are also missing when saving as a prefab:
```
(
  resources: {},
  entities: {
    25769803785: (
      components: {
        "bevy_transform::components::transform::Transform": (
          translation: (
            x: 0.0,
            y: 0.0,
            z: 0.0,
          ),
          rotation: (
            x: -0.7071068,
            y: 0.0,
            z: 0.0,
            w: 0.7071067,
          ),
          scale: (
            x: 100.0,
            y: 100.0,
            z: 100.0,
          ),
        ),
        "bevy_render::view::visibility::Visibility": Inherited,
        "space_prefab::save::ChildrenPrefab": ([
          4294967308,
          4294967307,
          8589934600,
        ]),
      },
    ),
    30064771082: (
      components: {
        "bevy_transform::components::transform::Transform": (
          translation: (
            x: 0.0,
            y: 0.0,
            z: 0.0,
          ),
          rotation: (
            x: 0.0,
            y: 0.0,
            z: 0.0,
            w: 1.0,
          ),
          scale: (
            x: 1.0,
            y: 1.0,
            z: 1.0,
          ),
        ),
        "bevy_render::view::visibility::Visibility": Inherited,
        "space_prefab::save::ChildrenPrefab": ([
          25769803785,
        ]),
      },
    ),
  },
)
```

**Comment by rewin123** [comment]
Thanks for the bug report! Will try to fix today

**Comment by rewin123** [comment]
Fixed at #152
